frontend/public/components/custom-resource-definition.tsx

```diff
@@ -107,6 +114,53 @@ const Established: React.FC<{ crd: CustomResourceDefinitionKind }> = ({ crd }) =
   );
 };
 
+const EmptyVersionsMsg: React.FC<{}> = () => <EmptyBox label="CRD Versions" />;
+
+const CRDVersionsHeader = () => [
+  {
+    title: 'Name',
+    sortField: 'name',
```

Comment:
I don't think sorting will work as expected here. You would need to use the `crdVersionSort` func you wrote in order for this to sort correctly.

Reply:
Spoke with @jcaianirh and it seems that this has already been discussed and a fix would be beyond the scope of this PR. Resolving.